docs/features/pull_rate_limiting.md

```diff
@@ -0,0 +1,37 @@
+# Image Registry rate limiting
```

Comment:
This page is still WIP

Reply:
Reduced in content to this current form, which I think is just enough for this PR.